packages/standards/biome-esm.jsonc

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "./node_modules/@biomejs/biome/configuration_schema.json",
+  "linter": {
+    "domains": {
+      "react": "all",
+    },
+    "rules": {
+      "nursery": {
+        "noGlobalDirnameFilename": "error",
```

Comment:
Previously enabled in `@readme/eslint-config/esm` with the help of [unicorn/prefer-module](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/prefer-module.md).